### PR TITLE
Version 3.0

### PR DESCRIPTION
**Pull request by alex-ju**
### :pushpin: Summary

This is a PR to group changes for the next major release, [version 3.0.0](https://github.com/hashicorp/design-system/milestone/1?closed=1).

### CODE REVIEW

**Comment by alex-ju** [comment]
Rebased this PR to main to resolve some recent conflicts

File: packages/components/package.json
Comment (aklkv) [inline packages/components/package.json:56]: this package is entering maintainece mode and is no longer the recommended way to use TS https://github.com/typed-ember/ember-cli-typescript/issues/1568

**Comment by alex-ju** [comment]
Currently failing the test matrix with 
Screenshot 2023-10-24 at 12 14 02

I have double checked and the specific versions of these actions are in TSCCR ([checkout](https://github.com/hashicorp/security-tsccr/blob/main/components/github_actions/actions-checkout.hcl) and [setup-node](https://github.com/hashicorp/security-tsccr/blob/main/components/github_actions/actions-setup-node.hcl)) so I'm a bit puzzled.

**Review by didoo — APPROVED** [review]
@alex-ju it's practically impossible to re-review everything, I am approving after a high level check.

What I've done is to push a commit to update the `changeset` entries to the new format.

**Comment by alex-ju** [comment]
> @alex-ju it's practically impossible to re-review everything, I am approving after a high-level check.

Indeed, more of a formality so I can merge this as an author.

> What I've done is to push a commit to update the `changeset` entries to the new format.

Ah, thanks for that! I haven’t forgotten about them, was thinking I can edit them in bulk in `hashibot-hds`'s PR

I'm seeing a conflict in the `yarn.lock` after the recent merge – will resolve than then merge

**Pull request merged**
